@here/harp-examples/src/bounds-generation.ts

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
Can you add globe and wrap around options to the example please. (Globe is available in MapControlsUI already)